### ISSUE TITLE
ModuleNotFoundError: No module named 'vivisect.impapi.windows.arm'

### ISSUE THREAD

**Issue by williballenthin** [vivisect/vivisect#423]
```
> python vivisect/vivbin.py --bulk samples/0391b7a9407757a5efb72cff837509bc07635da64eb5645d3aa3284052287c17                                                                            working?
Traceback (most recent call last):
  File "vivisect/vivbin.py", line 144, in <module>
    main()
  File "vivisect/vivbin.py", line 98, in main
    vw.loadFromFile(fname, fmtname=args.parsemod)
  File "/home/user/code/vivisect/vivisect/__init__.py", line 2730, in loadFromFile
    self._snapInAnalysisModules()
  File "/home/user/code/vivisect/vivisect/base.py", line 197, in _snapInAnalysisModules
    viv_analysis.addAnalysisModules(self)
  File "/home/user/code/vivisect/vivisect/analysis/__init__.py", line 42, in addAnalysisModules
    vw.addImpApi('windows','arm')
  File "/home/user/code/vivisect/vivisect/impapi/__init__.py", line 66, in addImpApi
    __import__(modname)
ModuleNotFoundError: No module named 'vivisect.impapi.windows.arm'
```

is there a file missing here? https://github.com/vivisect/vivisect/tree/v1.0.3/vivisect/impapi/windows

source: https://github.com/fireeye/capa/issues/706
sample: https://www.virustotal.com/gui/file/0391b7a9407757a5efb72cff837509bc07635da64eb5645d3aa3284052287c17/detection

**Comment by rakuy0** [comment]
Yea, I'd want to know if @atlas0fd00m has this file floating around on his box and just forgot to commit it before just removing the ref.

**Comment by atlas0fd00m** [comment]
for review: https://github.com/vivisect/vivisect/pull/431

**Comment by williballenthin** [comment]
closed in #431
